### DISCUSSION TITLE
What's missing for metrics API ?

### DISCUSSION THREAD

**Discussion by jvanz** [open-telemetry/opentelemetry-go#2340] [Q&A] [answered]
Hi guys! I'm working on integrating OpenTelemetry in a golang project. AFAICS in the docs, the API is in Alpha. My question what is missing to move this API to stable?

**Comment by MrAlias** [comment, marked as answer]
Thanks for your interest in the project! [A great overview](https://docs.google.com/presentation/d/1o4ycmdh-G1EnQH0sonn4K4izf0zKC28ZZN5GjT6AUUQ/edit#slide=id.p) of the current state of metrics in the project was given [in our last SIG meeting](https://youtu.be/m6uN5Zk5BNA?t=310). That contains not only our current state, but things we need to accomplish to release.

Important to also note is the OpenTelemetry metric specification has just reached "Feature Freeze" status. We are not able to release the metric signal prior to the specification becoming stable. Though, it may be some time after that before we are able to successfully do so.

**Reply by jvanz** [reply]
Thanks for the links @MrAlias! I'm getting used to the project and it would be cool to help the project on this. I'll take a look in the issues and see if I can to do something. Besides this,  I'm able to send some metrics to OpenTelemetry collector taking a look in the example and tests.
